corpdir-srv-sync/src/main/java/com/github/bordertech/corpdir/sync/common/DataSynchronisation.java

```diff
@@ -0,0 +1,23 @@
+package com.github.bordertech.corpdir.sync.common;
+
+import com.github.bordertech.corpdir.api.response.DataResponse;
+import java.util.List;
+
+/**
+ *

```

Comment:
Class description? and Parameter descriptions?

Reply:
This is just a first cut so haven't added javadoc to every class, anyway I have done now